<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelecaoPJWindow</class>
 <widget class="QMainWindow" name="SelecaoPJWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelecaoPJWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditArq_1"/>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBoxExibicao">
      <property name="title">
       <string>Exibição</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioButtonExbArq_1">
         <property name="text">
          <string>Exibir Arquivo 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonExbArq_2">
         <property name="text">
          <string>Exibir Arquivo 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonExbArqAmbos">
         <property name="text">
          <string>Exibir Ambos</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonSelecionar_2">
      <property name="text">
       <string>Selecionar ...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelArq_2">
      <property name="text">
       <string>Arquivo 2:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditArq_2"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButtonSelecionar_1">
      <property name="text">
       <string>Selecionar ...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBoxPreview">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditConteudo"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelArq_1">
      <property name="text">
       <string>Arquivo 1:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ordenação</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButtonOrdOcorr">
         <property name="text">
          <string>Ocorrências</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOrdLexic">
         <property name="text">
          <string>Lexicográfica</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
